polylogue/cli/app.py

```
@@ -92,6 +152,141 @@ def _legacy_candidates(root: Path) -> List[Path]:
+                    return
+                fi
+            fi
+            _describe 'values' completions
+        }
+
+        compdef _polylogue_complete polylogue
+        """
+    ).strip()
+
+
+def run_completions_cli(args: argparse.Namespace, env: CommandEnv) -> None:
```

Ruff (0.15.9)

[warning] Unused function argument: `env`

(ARG001)